client/src/main/java/io/getstream/chat/android/client/models/Attachment.kt

```diff
@@ -30,8 +32,25 @@ data class Attachment(
     var name: String? = null,
     var fallback: String? = null,
 
+    @IgnoreSerialisation
+    @IgnoreDeserialisation
+    var upload: File? = null,
+
+    @IgnoreSerialisation
+    @IgnoreDeserialisation
+    var uploadState: UploadState? = null,
```

Comment:
I'd be better to avoid null here. We could use a null object pattern and a default state could be defined called `object Unknown/Empty: UploadState()`